From ab05c0465470fbd5c7037404548ad66e594f5a84 Mon Sep 17 00:00:00 2001
From: Christian Bruel <christian.bruel@foss.st.com>
Date: Wed, 8 Oct 2025 11:32:17 +0200
Subject: [PATCH] arm64: dts: st: add msi-map in pcie_ep node

Add msi-map to map one PCIe device function MSI doorbell.

Signed-off-by: Christian Bruel <christian.bruel@foss.st.com>
Change-Id: Ic10dbd174c056fc1dd4ab842f2ad7a4b03193f8e
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/497596
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp251.dtsi | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/boot/dts/st/stm32mp251.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp251.dtsi
@@ -2396,6 +2396,7 @@
 				reset-names = "pcie";
 				phys = <&combophy PHY_TYPE_PCIE>;
 				phy-names = "pcie-phy";
+				msi-map = <0x0 &v2m0 0x0 0x1>;
 				access-controllers = <&rifsc 68>;
 				power-domains = <&d1_pd>;
 				status = "disabled";
